processing/src/main/java/org/apache/druid/query/rowsandcols/concrete/ColumnBasedFrameRowsAndColumns.java

```diff
@@ -44,12 +47,27 @@ public class ColumnBasedFrameRowsAndColumns implements RowsAndColumns, AutoClose
   private final RowSignature signature;
   private final LinkedHashMap<String, Column> colCache = new LinkedHashMap<>();
 
-  public ColumnBasedFrameRowsAndColumns(Frame frame, RowSignature signature)
+  @JsonCreator
```

Comment:
why this is needed? it doesn't seem to be used

Reply:
removed